timesketch/api/v1/resources/__init__.py

```diff
@@ -157,13 +157,29 @@ class ResourceMixin(object):
         'updated_at': fields.DateTime
     }
 
+    graph_fields = {
+        'id': fields.Integer,
+        'name': fields.String,
+        'user': fields.Nested(user_fields),
+        'created_at': fields.DateTime,
+        'updated_at': fields.DateTime
+    }
+
+    graphcache_fields = {
+        'id': fields.Integer,
+        'graph_elements': fields.String,
+        'created_at': fields.DateTime,
+        'updated_at': fields.DateTime
+    }
+
     sketch_fields = {
         'id': fields.Integer,
         'name': fields.String,
         'description': fields.String,
         'user': fields.Nested(user_fields),
         'timelines': fields.List(fields.Nested(timeline_fields)),
         'stories': fields.List(fields.Nested(story_fields)),
+        'graphs': fields.List(fields.Nested(graph_fields)),
```

Comment:
do we not want to have a list of the cached graphs in here as well?

Reply:
No, we don't render them anywhere except in the component and that one is fetching it on demand.